extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -17,6 +17,25 @@ public static function getIcon( $service ) {
 		return $icon;
 	}
 
+	/**
+	 * Get language for Page Share service.
+	 * For Anon users use the browser language.
+	 * For logged in user use user's language.
+	 * Both values can be overwritten by ?uselang parameter.
+	 *
+	 * @return String language
+	 */
+	public static function getLangForPageShare() {
+		global $wgLang, $wgRequest, $wgUser;
+
+		if ( $wgUser->isAnon() ) {
+			$lang = substr($_SERVER['HTTP_ACCEPT_LANGUAGE'], 0, 2);
```

Comment:
What if `$_SERVER['HTTP_ACCEPT_LANGUAGE']` is empty? Let's put some default here.